.env.staging

```diff
@@ -1,3 +1,5 @@
 REACT_APP_STREAMR_API_URL=https://staging.streamr.com/api/v1
+REACT_APP_STREAMR_WS_URL=ws://staging.streamr.com/api/v1/ws
```

Comment:
Might be better to throw out this staging conf?

Reply:
Yes. I think we should get rid of all .env files as https://github.com/motdotla/dotenv#faq states. We could have a .env.template like in `streamr-platform`. Working with .env files gets complicated when deploying to different environments.